### DISCUSSION TITLE
Weather forecast for next 3h 6h 9h in single screen

### DISCUSSION THREAD

**Discussion by ra-kal** [Blackymas/NSPanel_HA_Blueprint#1228] [Ideas]
Hi,

My kids liked the weather forecast on Google Nest thermostat. Every morning they looked at the display and they new, how the temperatures will be that day. It shown current values + forecast for next +3h +6h +9h. It was simple and effective.

Can we have such function on the NS Panel?
I can't imagine, that somebody will check forecast for next days on the panel. Usually we do it on the smart phone. But a completely different situation is to quickly check the forecast for example when leaving home for some activities. The hour forecast would be too short for a day, so the 3h forecast seems to most convenient.

**Comment by edwardtfn** [comment]
Moved to #1233

**Comment by edwardtfn** [comment]
- #399 
- #426 
- #831 
- #1228 
- #1233
